.github/workflows/cd.yaml

```diff
@@ -116,9 +104,28 @@ jobs:
               run: npm run smoke:headless
               timeout-minutes: 10
 
-            - name: Fetch AWS Credentials for Smoke Test
```

Comment:
In a previous change, we would fetch the credentials before every smoke test because the CJS and ES smoke tests are long running and creds used to time out before test finished. 

If we are removing the fetch before every test, we may want to increase the duration that the temporary credentials are valid when they are initially fetched.

Reply:
Fixed -- we now re-fetch credentials before running each smoke test (ES, CJS and CDN).